### ISSUE TITLE
Suggestion for a list of entities ignored by default

### ISSUE THREAD

**Issue by X-Machiavelli** [frikinjay/mob-stacker#5]
What do you think about excluding all villagers through a preset for smooth transactions with villagers. they stack up by default and I have to exclude them one by one, which is very inconvenient.
this mod is cool. thanks.

**Comment by frikinjay** [comment]
I do have entities in the default ignored entities config, I can add villagers as well. I'll pin this so people can suggest more entities to be added from other mods.

**Comment by X-Machiavelli** [comment]
Thanks for listening. but my intention is that I want to keep the unemployed villagers stacked and only remove the employed ones. If I'm playing a modpack, there are a lot of tradable entities and I think it would be difficult to remove them one by one (even if it's configurable). I don't know much about programming, is it possible to remove only tradable entities in bulk? your mod is cool enough, but I don't think it takes into account the trading system.

**Comment by frikinjay** [comment]
By default only unemployed villagers stack and only if they're of the same variant.

Same goes for frogs, foxes, mooshrooms, axolots and wolves. They only stack if of the similar variant.

**Comment by NinjaGamer4225** [comment]
is there a way for the Trial Chamber mobs to not stack

**Comment by frikinjay** [comment]
@NinjaGamer4225 yes, you can add a tag to the mobs before they spawn, not sure if this is achievable through datapacks but I can add this in the config in the future.
